docs/source/userguide/command_line.rst

```diff
@@ -1,508 +1,520 @@
 .. dropdown:: Distribution Statement
 
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
+ This source code is protected under the license referenced at
+ https://github.com/NRLMMD-GEOIPS.
 
 .. _command_line:
 
-Command Line Interface
-**********************
+Command Line Interface (CLI)
+****************************
 
-.. _geoips:
+.. warning::
 
-:ref:`GeoIPS CLI <geoips>`
+    The CLI is currently under development.
+    The CLI may change without warning!
+    Please consult this documentation for up-to-date info on the CLI.
 
-The GeoIPS Command Line Interface (CLI) is currently under development, but we'll keep
-updated instructions in this documentation that reflect the current state of the CLI.
+The CLI interacts with GeoIPS artifacts.
+A GeoIPS artifact is a piece of data or functionality,
+such as an Interface, Package, Plugin,
+Family, Test Script, Test Dataset, etc.
 
-The GeoIPS CLI is used to interrogate / utilize GeoIPS ``artifacts`` in a user-friendly
-manner. A ``GeoIPS artifact`` is an term used to reflect the chosen piece of data or
-functionality implemented in GeoIPS. This could be a GeoIPS Interface, Package, Plugin,
-Family, Test Script, Test Dataset, etc. that is used and/or implemented throughout the
-GeoIPS Environment.
+CLI commands are split up into two groups by their utility:
 
-The CLI is split up into two groups as of right now. CLI commands which
-:ref:`retrieve information<information_retrieval>` from GeoIPS and CLI commands which
-:ref:`perform processes<performing_processes>` via GeoIPS. We'll dive into each of those
-sections now.
+ - `Information retrieval`_ commands
```

Comment:
I'm not sure that I like "Information retrieval commands" or "Action performing commands". Ideas:

### "Information retrieval commands" alternatives
"Passive commands"
"Discovery commands"
"Introspection commands"
"Introspective commands"

### "Action performing commands" alternatives
"Active commands"
"Action commands"
"Operational commands"
"Execution commands"

Reply:
I like "Discovery Commands" and "Action commands"